### PR TITLE
update monthly images, add 6.9 base container

### PR DESCRIPTION
**Pull request by jperrin**

### CODE REVIEW

File: library/centos [hunk 1]
```diff
@@ -5,11 +5,11 @@ Constraints: !aufs
...
 GitFetch: refs/heads/CentOS-7
-GitCommit: 3463e5f011fbd82f8283757408427802ee020ee4
+GitCommit: 50281d86d6ed5c61975971150adfd0ede86423bb
 
 Tags: centos6, 6
 GitFetch: refs/heads/CentOS-6
-GitCommit: 697f0fae0ff2b942c37eb7f4100163c1e5d3e922
+GitCommit: 5048dc2d18390e22bc9c95fe7526f7f63490b7b6
 
 Tags: centos5, 5
```
Comment (tianon) [inline library/centos]: Isn't 5 EOL now? :smile: :innocent:

For context, removing the tag here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

**Comment by yosifkit** [comment]
Diff:

<details>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index fcc2485..fd36668 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,6 +4,7 @@ centos:6
 centos:6.6
 centos:6.7
 centos:6.8
+centos:6.9
 centos:7
 centos:7.0.1406
 centos:7.1.1503
@@ -15,6 +16,7 @@ centos:centos6
 centos:centos6.6
 centos:centos6.7
 centos:centos6.8
+centos:centos6.9
 centos:centos7
 centos:centos7.0.1406
 centos:centos7.1.1503
diff --git a/centos_6/Dockerfile b/centos_6.9/Dockerfile
similarity index 60%
copy from centos_6/Dockerfile
copy to centos_6.9/Dockerfile
index 750674d..7321aab 100644
--- a/centos_6/Dockerfile
+++ b/centos_6.9/Dockerfile
@@ -1,10 +1,9 @@
 FROM scratch
-MAINTAINER https://github.com/CentOS/sig-cloud-instance-images
 ADD centos-6-docker.tar.xz /
 
 LABEL name="CentOS Base Image" \
     vendor="CentOS" \
     license="GPLv2" \
-    build-date="20170315"
+    build-date="20170406"
 
 CMD ["/bin/bash"]
diff --git a/centos_6/centos-6-docker.tar.xz b/centos_6.9/centos-6-docker.tar.xz
similarity index 76%
copy from centos_6/centos-6-docker.tar.xz
copy to centos_6.9/centos-6-docker.tar.xz
index 9d89ea2..d7aa25f 100644
Binary files a/centos_6/centos-6-docker.tar.xz and b/centos_6.9/centos-6-docker.tar.xz differ
diff --git a/centos_6/centos-6-docker.tar.xz  'tar -t' b/centos_6.9/centos-6-docker.tar.xz  'tar -t'
similarity index 99%
copy from centos_6/centos-6-docker.tar.xz  'tar -t'
copy to centos_6.9/centos-6-docker.tar.xz  'tar -t'
index 6507b28..b1671be 100644
--- a/centos_6/centos-6-docker.tar.xz  'tar -t'	
+++ b/centos_6.9/centos-6-docker.tar.xz  'tar -t'	
@@ -223,6 +223,8 @@
 ./etc/pki/nssdb/key4.db
 ./etc/pki/nssdb/pkcs11.txt
 ./etc/pki/nssdb/secmod.db
+./etc/pki/nss-legacy/
+./etc/pki/nss-legacy/nss-rhel6.config
 ./etc/pki/rpm-gpg/
 ./etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
 ./etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Debug-6
@@ -1211,10 +1213,9 @@
 ./usr/lib64/krb5/plugins/
 ./usr/lib64/krb5/plugins/authdata/
 ./usr/lib64/krb5/plugins/kdb/
-./usr/lib64/krb5/plugins/kdb/db2.so
 ./usr/lib64/krb5/plugins/libkrb5/
 ./usr/lib64/krb5/plugins/preauth/
-./usr/lib64/libbfd-2.20.51.0.2-5.44.el6.so
+./usr/lib64/libbfd-2.20.51.0.2-5.46.el6.so
 ./usr/lib64/libbind9.so.80
 ./usr/lib64/libbind9.so.80.0.4
 ./usr/lib64/libcrack.so.2
@@ -1264,10 +1265,6 @@
 ./usr/lib64/libisccfg.so.82.0.1
 ./usr/lib64/libisc.so.83
 ./usr/lib64/libisc.so.83.0.3
-./usr/lib64/libkadm5clnt_mit.so.8
-./usr/lib64/libkadm5clnt_mit.so.8.0
-./usr/lib64/libkadm5srv_mit.so.8
-./usr/lib64/libkadm5srv_mit.so.8.0
 ./usr/lib64/libkdb5.so.6
 ./usr/lib64/libkdb5.so.6.0
 ./usr/lib64/liblua-5.1.so
@@ -1291,7 +1288,7 @@
 ./usr/lib64/libnsspem.so
 ./usr/lib64/libnsssysinit.so
 ./usr/lib64/libnssutil3.so
-./usr/lib64/libopcodes-2.20.51.0.2-5.44.el6.so
+./usr/lib64/libopcodes-2.20.51.0.2-5.46.el6.so
 ./usr/lib64/libp11-kit.so.0
 ./usr/lib64/libp11-kit.so.0.0.0
 ./usr/lib64/libpanel.so.5
@@ -8001,7 +7998,6 @@
 ./usr/share/zoneinfo/America/Port_of_Spain
 ./usr/share/zoneinfo/America/Porto_Velho
 ./usr/share/zoneinfo/America/Puerto_Rico
-./usr/share/zoneinfo/America/Punta_Arenas
 ./usr/share/zoneinfo/America/Rainy_River
 ./usr/share/zoneinfo/America/Rankin_Inlet
 ./usr/share/zoneinfo/America/Recife
@@ -8590,7 +8586,6 @@
 ./usr/share/zoneinfo/posix/America/Port_of_Spain
 ./usr/share/zoneinfo/posix/America/Porto_Velho
 ./usr/share/zoneinfo/posix/America/Puerto_Rico
-./usr/share/zoneinfo/posix/America/Punta_Arenas
 ./usr/share/zoneinfo/posix/America/Rainy_River
 ./usr/share/zoneinfo/posix/America/Rankin_Inlet
 ./usr/share/zoneinfo/posix/America/Recife
@@ -9207,7 +9202,6 @@
 ./usr/share/zoneinfo/right/America/Port_of_Spain
 ./usr/share/zoneinfo/right/America/Porto_Velho
 ./usr/share/zoneinfo/right/America/Puerto_Rico
-./usr/share/zoneinfo/right/America/Punta_Arenas
 ./usr/share/zoneinfo/right/America/Rainy_River
 ./usr/share/zoneinfo/right/America/Rankin_Inlet
 ./usr/share/zoneinfo/right/America/Recife
@@ -9652,7 +9646,6 @@
 ./usr/share/zoneinfo/UTC
 ./usr/share/zoneinfo/WET
 ./usr/share/zoneinfo/W-SU
-./usr/share/zoneinfo/zone1970.tab
 ./usr/share/zoneinfo/zone.tab
 ./usr/share/zoneinfo/Zulu
 ./usr/src/
diff --git a/centos_6/Dockerfile b/centos_6/Dockerfile
index 750674d..7321aab 100644
--- a/centos_6/Dockerfile
+++ b/centos_6/Dockerfile
@@ -1,10 +1,9 @@
 FROM scratch
-MAINTAINER https://github.com/CentOS/sig-cloud-instance-images
 ADD centos-6-docker.tar.xz /
 
 LABEL name="CentOS Base Image" \
     vendor="CentOS" \
     license="GPLv2" \
-    build-date="20170315"
+    build-date="20170406"
 
 CMD ["/bin/bash"]
diff --git a/centos_6/centos-6-docker.tar.xz b/centos_6/centos-6-docker.tar.xz
index 9d89ea2..bfe869d 100644
Binary files a/centos_6/centos-6-docker.tar.xz and b/centos_6/centos-6-docker.tar.xz differ
diff --git a/centos_6/centos-6-docker.tar.xz  'tar -t' b/centos_6/centos-6-docker.tar.xz  'tar -t'
index 6507b28..5e51de8 100644
--- a/centos_6/centos-6-docker.tar.xz  'tar -t'	
+++ b/centos_6/centos-6-docker.tar.xz  'tar -t'	
@@ -223,6 +223,8 @@
 ./etc/pki/nssdb/key4.db
 ./etc/pki/nssdb/pkcs11.txt
 ./etc/pki/nssdb/secmod.db
+./etc/pki/nss-legacy/
+./etc/pki/nss-legacy/nss-rhel6.config
 ./etc/pki/rpm-gpg/
 ./etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
 ./etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Debug-6
@@ -1211,10 +1213,9 @@
 ./usr/lib64/krb5/plugins/
 ./usr/lib64/krb5/plugins/authdata/
 ./usr/lib64/krb5/plugins/kdb/
-./usr/lib64/krb5/plugins/kdb/db2.so
 ./usr/lib64/krb5/plugins/libkrb5/
 ./usr/lib64/krb5/plugins/preauth/
-./usr/lib64/libbfd-2.20.51.0.2-5.44.el6.so
+./usr/lib64/libbfd-2.20.51.0.2-5.46.el6.so
 ./usr/lib64/libbind9.so.80
 ./usr/lib64/libbind9.so.80.0.4
 ./usr/lib64/libcrack.so.2
@@ -1264,10 +1265,6 @@
 ./usr/lib64/libisccfg.so.82.0.1
 ./usr/lib64/libisc.so.83
 ./usr/lib64/libisc.so.83.0.3
-./usr/lib64/libkadm5clnt_mit.so.8
-./usr/lib64/libkadm5clnt_mit.so.8.0
-./usr/lib64/libkadm5srv_mit.so.8
-./usr/lib64/libkadm5srv_mit.so.8.0
 ./usr/lib64/libkdb5.so.6
 ./usr/lib64/libkdb5.so.6.0
 ./usr/lib64/liblua-5.1.so
@@ -1291,7 +1288,7 @@
 ./usr/lib64/libnsspem.so
 ./usr/lib64/libnsssysinit.so
 ./usr/lib64/libnssutil3.so
-./usr/lib64/libopcodes-2.20.51.0.2-5.44.el6.so
+./usr/lib64/libopcodes-2.20.51.0.2-5.46.el6.so
 ./usr/lib64/libp11-kit.so.0
 ./usr/lib64/libp11-kit.so.0.0.0
 ./usr/lib64/libpanel.so.5
diff --git a/centos_7/Dockerfile b/centos_7/Dockerfile
index 38d92bc..5f40812 100644
--- a/centos_7/Dockerfile
+++ b/centos_7/Dockerfile
@@ -1,10 +1,9 @@
 FROM scratch
-MAINTAINER https://github.com/CentOS/sig-cloud-instance-images
 ADD centos-7-docker.tar.xz /
 
 LABEL name="CentOS Base Image" \
     vendor="CentOS" \
     license="GPLv2" \
-    build-date="20170315"
+    build-date="20170406"
 
 CMD ["/bin/bash"]
diff --git a/centos_7/centos-7-docker.tar.xz b/centos_7/centos-7-docker.tar.xz
index f9f816d..a5fffa7 100644
Binary files a/centos_7/centos-7-docker.tar.xz and b/centos_7/centos-7-docker.tar.xz differ
diff --git a/centos_7/centos-7-docker.tar.xz  'tar -t' b/centos_7/centos-7-docker.tar.xz  'tar -t'
index 6d39e5c..49f4050 100644
--- a/centos_7/centos-7-docker.tar.xz  'tar -t'	
+++ b/centos_7/centos-7-docker.tar.xz  'tar -t'	
@@ -341,7 +341,7 @@
 ./tmp/
 ./tmp/.font-unix/
 ./tmp/.ICE-unix/
-./tmp/ks-script-AkSWCX
+./tmp/ks-script-MeHyWm
 ./tmp/.Test-unix/
 ./tmp/.X11-unix/
 ./tmp/.XIM-unix/
```

</details>

**Comment by yosifkit** [comment]
Build test of #2832; c0a05cce (`centos`):

```console

$ bashbrew build centos:latest
Building bashbrew/cache:38f70e39f1c4e355c5b76fc5410cb089cafe3a071505169e617e7945fc3ab052 (centos:latest)
Tagging centos:latest
Tagging centos:centos7
Tagging centos:7

$ test/run.sh centos:latest
testing centos:latest
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos6
Building bashbrew/cache:2f17c3fea884e29bd09a9c78b984cb6cc05fc65389636c789ea8a3c13d5cbda1 (centos:centos6)
Tagging centos:centos6
Tagging centos:6

$ test/run.sh centos:centos6
testing centos:centos6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos5
Using bashbrew/cache:f65ad6a5052b386734f84ccccf2c25a733431eeae988140baad1e5399bdce812 (centos:centos5)
Tagging centos:centos5
Tagging centos:5

$ test/run.sh centos:centos5
testing centos:centos5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...error: crypt password detected for 'root': '$1$UKLtvLuY$kka6S665oCFmU7ivSDZzU.'
failed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos7.3.1611
Using bashbrew/cache:61622958167fbac7661e35df6ba5fc3a081d8034255f3c26105be427e250ddba (centos:centos7.3.1611)
Tagging centos:centos7.3.1611
Tagging centos:7.3.1611

$ test/run.sh centos:centos7.3.1611
testing centos:centos7.3.1611
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos7.2.1511
Using bashbrew/cache:6e0a78389c5bef1c69207f935212b4d8d8e98d6540337a25545d85eca86e43e7 (centos:centos7.2.1511)
Tagging centos:centos7.2.1511
Tagging centos:7.2.1511

$ test/run.sh centos:centos7.2.1511
testing centos:centos7.2.1511
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos7.1.1503
Using bashbrew/cache:af64f05f3de65c78d5e75ab055996c3ac6e3aec49d9001ef05032c999208e5ed (centos:centos7.1.1503)
Tagging centos:centos7.1.1503
Tagging centos:7.1.1503

$ test/run.sh centos:centos7.1.1503
testing centos:centos7.1.1503
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...error: crypt password detected for 'root': '$1$UKLtvLuY$kka6S665oCFmU7ivSDZzU.'
failed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos7.0.1406
Using bashbrew/cache:b675060f29caa985d4069b0d7e7f5b5a123461f1e931ebc37d9f4c0cd07b0840 (centos:centos7.0.1406)
Tagging centos:centos7.0.1406
Tagging centos:7.0.1406

$ test/run.sh centos:centos7.0.1406
testing centos:centos7.0.1406
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...shellshock_test.sh: line 17:    15 Segmentation fault      shellshocker="() { x() { _;}; x() { _;} <<a; }" bash -c date 2> /dev/null
shellshock_test.sh: line 50:    32 Segmentation fault      bash -c 'true <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF' 2> /dev/null
failed
	'no-hard-coded-passwords' [3/4]...error: crypt password detected for 'root': '$1$UKLtvLuY$kka6S665oCFmU7ivSDZzU.'
failed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos6.8
Using bashbrew/cache:9775568854030068d0879e45b370be8a29524eb91c0d2fecfbbd69ba4d415a1d (centos:centos6.8)
Tagging centos:centos6.8
Tagging centos:6.8

$ test/run.sh centos:centos6.8
testing centos:centos6.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos6.9
Building bashbrew/cache:259c8c21ccd9f3f15a89b681b167965cea1c911236286c9885ad6c36e63ad722 (centos:centos6.9)
Tagging centos:centos6.9
Tagging centos:6.9

$ test/run.sh centos:centos6.9
testing centos:centos6.9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos6.7
Using bashbrew/cache:788235800c44491534389f1742c1f7352745d43a49c6876639a229597a4d9c61 (centos:centos6.7)
Tagging centos:centos6.7
Tagging centos:6.7

$ test/run.sh centos:centos6.7
testing centos:centos6.7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...error: crypt password detected for 'root': '$6$QhN6G8YM$LDd5zhqarhgMy6/e1c6wYwbusi4RZBz3lfRNQ1p5VquqtHzIj.Tf9r7cqoaSLgI3FiCPzfePyTBG7omBKv9bF0'
failed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos6.6
Using bashbrew/cache:efa70939a6e829a4a9f3b0267d381abcf417a1292ee3d3c641b239f731c76d2e (centos:centos6.6)
Tagging centos:centos6.6
Tagging centos:6.6

$ test/run.sh centos:centos6.6
testing centos:centos6.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build centos:centos5.11
Using bashbrew/cache:c0e7b43e54525b0a996eb831741ba2e80a77aea02baaae62b2d8c1f4432c796b (centos:centos5.11)
Tagging centos:centos5.11
Tagging centos:5.11

$ test/run.sh centos:centos5.11
testing centos:centos5.11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...shellshock_test.sh: line 17:    14 Segmentation fault      shellshocker="() { x() { _;}; x() { _;} <<a; }" bash -c date 2> /dev/null
shellshock_test.sh: line 50:    30 Segmentation fault      bash -c 'true <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF <<EOF' 2> /dev/null
bash: line 129: syntax error near `x129'
bash: line 129: `for x129 in ; do :'
failed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```


The following images failed at least one test:

- `centos`:
  - `centos:centos5`
  - `centos:centos7.1.1503`
  - `centos:centos7.0.1406`
  - `centos:centos6.7`
  - `centos:centos5.11`

File: library/centos (hunk 1 above)
Comment (jperrin) [inline library/centos]: ah, okay. I was under the impression removing it from the file here would remove it from the hub as well.

**Pull request merged**